ironic/drivers/modules/drac/raid.py

```diff
@@ -202,20 +271,77 @@ def delete_virtual_disk(node, virtual_disk):
         raise exception.DracOperationError(error=exc)
 
 
-def commit_config(node, raid_controller, reboot=False):
+def reset_config(node, raid_controller):
+    """Delete all virtual disk and unassign all hotspares physical disk
+
+    :param node: an ironic node object.
+    :param raid_controller: id of the RAID controller.
+    :returns: a dictionary containing the commit_needed key with a boolean
```

Comment:
In the line above, I believe that key that will be returned is: "is_commit_required"

Reply:
Yes i will change and update in new commit..i have just injected reset_config() method to test raid/jbod disks conversion..